recipes/holodeck-gw/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "holodeck-gw" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/holodeck-gw-{{ version }}.tar.gz
+  sha256: 6a20222953caaf22a98488a0de60a9cc74f97cc4ef912450ece4c5d02ff6a135
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools >=40.6.0
+    - wheel
+    - cython <3.0.0
+    - numpy <2.0.0
```

Just a test, I believe that this packages dependencies may be outdated and an older numpy may help. The "right" fix would be updating the feedstocks of the dependencies.

```suggestion
    - numpy 1.24.4
```